src/sage/symbolic/ring.pyx

```diff
@@ -862,6 +862,16 @@ cdef class SymbolicRing(sage.rings.abc.SymbolicRing):
             Traceback (most recent call last):
             ...
             ValueError: cannot specify n for multiple symbol names
+        
+        Check that :trac:`36833` is fixed: Variables whose domain is not
+        defined should be considered in complex plane as described in documentation::
```

Comment:
Could you point out where in documentation you see this " considered in complex plane" ?

Reply:
![Screenshot from 2023-12-10 19-07-00](https://github.com/sagemath/sage/assets/137716667/36a56b86-5ac6-46e9-8da9-55879b0fcd99)

Reply:
thanks!